bin/yaml/hook.yaml

```diff
@@ -8,3 +8,7 @@ compilers:
     dir: hook/hook-{name}
     targets:
       - 0.1.0
+    nightly:
+      if: nightly
+      type: nightlytarballs
+      install_always: true
```

Comment:
there's no `target:` list here so this doesn't do anything. Let me see if I can work out what the name is supposed to be

Reply:
Pinged you in slack: should I update to always install 0.1.0 as the "trunk" build? (we don't usually do this but it's up to you)